mappings/net/minecraft/resource/pack/CompositeResourcePack.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_hdtifwml net/minecraft/resource/pack/CompositeResourcePack
+	COMMENT A {@link ResourcePack} consisting of a primary resource pack and one or more overlays.
+	FIELD f_gyptkxaa primary Lnet/minecraft/unmapped/C_tguinuvn;
+	FIELD f_uchhhtst packStack Ljava/util/List;
```

Comment:
If it contains every packs:
```suggestion
	FIELD f_uchhhtst packs Ljava/util/List;
```

If it only contains overlays:
```suggestion
	FIELD f_uchhhtst overlays Ljava/util/List;
```

Reply:
I specifically call it stack because it is the *reverse* of the overlay list plus the primary

Reply:
Maybe worth javadoc'ing the constructor saying that packs are iterated like: `primary`, `lastOverlay`, ..., `firstOverlay`?